### PR TITLE
Feature/1.main layout

### PR DESCRIPTION
**Pull request by kappa3-3**

### CODE REVIEW

File: src/components/Calendar/style.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import '../../styles/colors.scss';
+@import '../../styles/mixins.scss';
+
+.calendar-container {
```
Comment (EmOlsz) [inline src/components/Calendar/style.scss]: Notice that 'container-' as part of class name is rewritten by you two times. Use ampersand (&) instead and avoid repeating your code

File: src/components/Calendar/style.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@import '../../styles/colors.scss';
+@import '../../styles/mixins.scss';
```
Comment (EmOlsz) [inline src/components/Calendar/style.scss]: 1. Create another file, "commons.scss", with all files that are being imported throughout the app. Repleace existing imports with "commons.scss" one
2. I see you are not using partials - colors.scss, mixins.scss and main.scss are three files. This is not optimized way of development, Sass gives us feature called partials (with prefix _). Refactor colors and mixins files. main.scss should be the only file, which stays as it is.

File: src/components/Main/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TaskList from '../TaskList/TaskList'
+import Calendar from '../Calendar/Calendar'
+import MenuList from '../MenuList/MenuList'
+
+import './style.scss';
+
+export default function Footer() {
```
Comment (EmOlsz) [inline src/components/Main/Main.jsx]: Probably you meant Main ;)

File: src/components/MenuList/MenuList.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+export default function MenuList () {
+
+return (
+    <div className='menuList-container'>
+            <div className='menuList-header'>
+                <span>Menu</span>
+            </div>
+            <div className='menuList-items'>
+                {/*<SingleMenu />*/}
```
Comment (EmOlsz) [inline src/components/MenuList/MenuList.jsx]: Do not commit code with comments - let's remove it for now

File: src/components/TaskList/TaskList.jsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    <div className='taskList-container'>
+
+            <div className='taskList-header'>
+                <span className='taskList-header-text'>Tasks</span>
+                <i className="material-icons plus">
+                    control_point
+                </i>
+            </div>
+            <div className='taskList-items'>
+                {/*<SingleTask />*/}
```
Comment (EmOlsz) [inline src/components/TaskList/TaskList.jsx]: Same as above

File: src/components/TaskList/TaskList.jsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+import './style.scss';
+
+export default function TaskList () {
+return (
+    <div className='taskList-container'>
+
+            <div className='taskList-header'>
+                <span className='taskList-header-text'>Tasks</span>
+                <i className="material-icons plus">
```
Comment (EmOlsz) [inline src/components/TaskList/TaskList.jsx]: Why are you not using material-ui Button or Icon components? We should use them as often as we need

File: src/components/TaskList/style.scss [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  flex-grow: 0.5;
+  @include element-container;
+}
+.taskList-header {
+  @include element-header;
+}
+.taskList-header-text {
+  width: 100%;
+}
+.plus {
```
Comment (EmOlsz) [inline src/components/TaskList/style.scss]: Let's be consequent and use BEM

File: src/styles/colors.scss [hunk 8]
```diff
@@ -0,0 +1,8 @@
+$gray: #646164;
+$gray-opacity: rgba(100, 97, 100, .6);
+$white-tone: rgba(238, 237, 238, 1);
```
Comment (EmOlsz) [inline src/styles/colors.scss]: If you are applying opacity equal to 1, then you don't opacity :) Let's replace it to hex

File: src/styles/main.scss [hunk 9]
```diff
@@ -0,0 +1,14 @@
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```
Comment (EmOlsz) [inline src/styles/main.scss]: Add separate variable $main-font and use it here. We might need it somewhere else one more time

File: src/styles/main.scss [hunk 10]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (EmOlsz) [inline src/styles/main.scss]: Remove this empty line

File: src/components/Footer/Footer.jsx [hunk 11]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import './style.scss';
+
+export default function Footer () {
+return (
```
Comment (EmOlsz) [inline src/components/Footer/Footer.jsx]: Please pay attention to those indents. Same goes for other files

File: src/components/Main/Main.jsx [hunk 12]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TaskList from '../TaskList/TaskList'
```
Comment (EmOlsz) [inline src/components/Main/Main.jsx]: Add one empty line between outer and local dependencies

File: src/components/MenuList/MenuList.jsx [hunk 13]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './style.scss';
+
+export default function MenuList () {
+
+return (
+    <div className='menuList-container'>
+            <div className='menuList-header'>
```
Comment (EmOlsz) [inline src/components/MenuList/MenuList.jsx]: Hmm, you use both camelCase and dash. menu-list__container is way better

File: src/components/TaskList/TaskList.jsx [hunk 14]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './style.scss';
+
+export default function TaskList () {
+return (
+    <div className='taskList-container'>
```
Comment (EmOlsz) [inline src/components/TaskList/TaskList.jsx]: Same as above

File: src/components/App.jsx [hunk 15]
```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 
+import Header from './Header/Header'
```
[the remark this answers is not in the document]
Comment (kappa3-3) [reply src/components/App.jsx]: #1 - done

File: src/components/Calendar/Calendar.jsx [hunk 16]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './style.scss';
+
+export default function Calendar () {
+return (
+    <div className='calendar-container'>
```
[the remark this answers is not in the document]
Comment (kappa3-3) [reply src/components/Calendar/Calendar.jsx]: #2 - done

File: src/components/Calendar/style.scss (hunk 2 above)
Comment (kappa3-3) [reply src/components/Calendar/style.scss]: #4 - done

File: src/styles/main.scss (hunk 10 above)
Comment (kappa3-3) [reply src/styles/main.scss]: done

File: src/components/Footer/Footer.jsx (hunk 11 above)
Comment (kappa3-3) [reply src/components/Footer/Footer.jsx]: done

File: src/components/MenuList/MenuList.jsx (hunk 13 above)
Comment (kappa3-3) [reply src/components/MenuList/MenuList.jsx]: #2 - done

File: src/components/TaskList/TaskList.jsx (hunk 14 above)
Comment (kappa3-3) [reply src/components/TaskList/TaskList.jsx]: #2 - done

File: src/components/Main/Main.jsx (hunk 12 above)
Comment (kappa3-3) [reply src/components/Main/Main.jsx]: done

File: src/components/Main/Main.jsx (hunk 3 above)
Comment (kappa3-3) [reply src/components/Main/Main.jsx]: done

File: src/components/MenuList/MenuList.jsx (hunk 4 above)
Comment (kappa3-3) [reply src/components/MenuList/MenuList.jsx]: done

File: src/components/TaskList/TaskList.jsx (hunk 5 above)
Comment (kappa3-3) [reply src/components/TaskList/TaskList.jsx]: done

File: src/components/TaskList/TaskList.jsx (hunk 6 above)
Comment (kappa3-3) [reply src/components/TaskList/TaskList.jsx]: done

File: src/components/TaskList/style.scss (hunk 7 above)
Comment (kappa3-3) [reply src/components/TaskList/style.scss]: done

File: src/styles/colors.scss (hunk 8 above)
Comment (kappa3-3) [reply src/styles/colors.scss]: done

File: src/styles/main.scss (hunk 9 above)
Comment (kappa3-3) [reply src/styles/main.scss]: done

**Pull request merged**